azure-kusto-data/tests/test_kusto_connection_string_builder.py

```diff
@@ -37,20 +37,18 @@ def test_aad_app(self):
         key = "key of application"
         kcsbs = [
             KustoConnectionStringBuilder(
-                "localhost;Application client Id={0};application Key={1};Authority Id={2} ; aad federated security = {3}".format(
-                    uuid, key, "microsoft.com", True
-                )
+                "localhost;Application client Id={};application Key={};Authority Id={} ; aad federated security = {}".format(uuid, key, "microsoft.com", True)
             ),
             KustoConnectionStringBuilder(
-                "Data Source=localhost ; Application Client Id={0}; Appkey ={1};Authority Id= {2} ; aad federated security = {3}".format(
+                "Data Source=localhost ; Application Client Id={}; Appkey ={};Authority Id= {} ; aad federated security = {}".format(

```

Comment:
nit: you are inconsistent with use of interpolation vs index free format
Ay reason for it?

Reply:
In some places the arguments were too complex to put inside an f-string (like if they have quotes inside of them),
 here specifically I didn't because I didn't want to transform the "microsoft.com" argument to be just part of the string